### PR TITLE
style updates

### PR DESCRIPTION
**Pull request by dbanksdesign**

### CODE REVIEW

File: .graphqlconfig.yml [hunk 1]
```diff
@@ -13,3 +13,6 @@ projects:
         region: us-east-1
         apiId: null
         maxDepth: 2
+extensions:
```
Comment (dbanksdesign) [inline .graphqlconfig.yml:16]: I don't know if this is needed? this happened when I did amplify push I think

File: src/components/Layout.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      {children}
+    </Flex>
+  );
+}
+
+
+export default function Layout({ children }: React.PropsWithChildren) {
+  const [colorMode, setColorMode] = React.useState<ColorMode>("light");
+  return (
+    <ThemeProvider theme={theme} colorMode={colorMode}>
```
Comment (dbanksdesign) [inline src/components/Layout.tsx:18]: Just need the ThemeProvider at the top of the render tree. If you want to use the theme somewhere you can just use `useTheme()`

File: src/components/Layout.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      <Flex direction="column" gap="0" width="100vw" height="100vh" backgroundColor="background.primary">
+        <AppHeader>
+          <Icon
```
Comment (dbanksdesign) [inline src/components/Layout.tsx:21]: Added the amplify logo here ;)

File: src/pages/index.tsx [hunk 4]
```diff
@@ -28,31 +32,20 @@ export default function Home() {
...
-      <Heading level={1}>Todo Form</Heading>
-      <Card borderRadius={"1rem"} padding="0">
+    <Flex direction="row" height="100%" width="100%" justifyContent="stretch" gap="0">
         <Flex
-          direction={"column"}
-          alignItems={"center"}
-          gap={"1rem"}
-          padding="1rem"
+          direction="column"
+          gap="medium"
```
Comment (dbanksdesign) [inline src/pages/index.tsx:42]: Used theme key values here

File: src/styles/globals.css [hunk 5]
```diff
@@ -1,27 +1,48 @@
-@tailwind base;
```
Comment (dbanksdesign) [inline src/styles/globals.css:1]: Removed default NextJS stuff and added a basic css reset

File: .graphqlconfig.yml (hunk 1 above)
Comment (ErikCH) [reply .graphqlconfig.yml:16]: 👍

File: src/components/Layout.tsx (hunk 2 above)
Comment (ErikCH) [reply src/components/Layout.tsx:18]: Good to know!

File: src/components/Layout.tsx (hunk 3 above)
Comment (ErikCH) [reply src/components/Layout.tsx:21]: 👍

**Pull request merged**